source/brailleDisplayDrivers/handyTech.py

```diff
@@ -103,6 +103,7 @@ def windowProc(self, hwnd: int, msg: int, wParam: int, lParam: int):
 MODEL_MODULAR_20 = b"\x80"
 MODEL_MODULAR_80 = b"\x88"
 MODEL_MODULAR_40 = b"\x89"
+MODEL_ACTIVATOR = b"\xA4"

```

Comment:
I thought the Activator supported HID. Is there something in the HandyTech protocol that isn't supported by your HID implementation?